### DISCUSSION TITLE
Tools to use, images for steps, dynamic ingredients calculator

### DISCUSSION THREAD

**Discussion by thehijacker** [mealie-recipes/mealie#1449] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Just discovered Mealie. Very simple to use even if import from Slovenian recepies site is not working. Bulk add wins here. I am missing this three features to make it best recepies self hosted service.

1. Among ingreadiens some recepies also include what tools will ne needed to make a meal. Like size of dishes, special tools, etc. I can not see any part of GUI where I could add this. Same as ingredients list for example. And example recipe: https://oblizniprste.si/slastna-kosila/najboljsa-bolonjska-omaka/
2. Usualy in between steps you can also have a picture how it should look up until this step. I can not paste directly the image. Would be nice if this would be also an option. Example recipe: https://oblizniprste.si/slastna-kosila/najboljsa-bolonjska-omaka/
3. I can add for how many portions is specific recipe. But if I decide to make smalle portion then the ingreadiens should also be automatically calculated. Like on this page: https://okusno.je/recept/salomejina-limonina-pita-z-meringo

![image](https://user-images.githubusercontent.com/13857043/175789565-d8c1d4fc-5594-4fd0-b330-1c43cc6141f8.png)

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
Pretty sure all of these features are available in v1. Have you looked at the demo site? demo.mealie.io.

**Reply by thehijacker** [reply]
Uf. I used the docker compose instructions to deploy it in my VM. None of this is included in that version. Will seek further on how to get latest version and check if this is indeed included. Sorry about this.

**Reply by hay-kot** [reply]
No worries, I had a wrong link in the ReadMe So lots of people have been confused lately. Checkout nightly.mealie.io for the beta docs.
